<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Asn1Acn::Internal::Libraries::SelectSourcePage</class>
 <widget class="QWidget" name="Asn1Acn::Internal::Libraries::SelectSourcePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import ASN.1 components</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="builtInRadio">
     <property name="text">
      <string>Built-in library</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="builtInCombo">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="customRadio">
     <property name="text">
      <string>Custom library</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Utils::PathChooser" name="asn1sccPathChooser">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QLineEdit</extends>
   <header>utils/pathchooser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>customRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>asn1sccPathChooser</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>builtInRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>builtInCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
